pkg/actions/param_set_test.go

```diff
@@ -54,6 +54,30 @@ func TestParamSet(t *testing.T) {
 	})
 }
 
+func TestParamSet_invalid_component(t *testing.T) {
+	withApp(t, func(appMock *amocks.App) {
+		componentName := "deployment"
+		path := "replicas"
+
+		in := map[string]interface{}{
+			OptionApp:   appMock,
+			OptionName:  componentName,
+			OptionPath:  path,
+			OptionValue: nil,
+		}
+
+		a, err := NewParamSet(in)
+		require.NoError(t, err)
+
+		a.resolvePathFn = func(app.App, string) (component.Module, component.Component, error) {
+			return nil, nil, nil
+		}
+
+		err = a.Run()
```

Comment:
This doesn't get as far as exercising the nil check  in `setLocal` since `OptionValue` is nil and fails to parse.

Reply:
Yeah... I updated the code

Reply:
Sweet!